### PR TITLE
Preview images for documents #3848

### PR DESCRIPTION
**Pull request by netzarbeiter**

### CODE REVIEW

File: docs/manual/article-management/content-elements/file-elements.en.md [hunk 1]
```diff
@@ -25,34 +34,95 @@ The special feature of Contao is that this download link also works with protect
 
 **Link title:** The link title is inserted as an `title` attribute in the HTML markup.
 
+
+{{< version "4.13" >}}
+
+### Preview settings
+
+{{% notice info %}}
+The PHP extension Imagick must be installed on the server to enjoy this feature.
```
Comment (ausi) [inline docs/manual/article-management/content-elements/file-elements.en.md]:
```suggestion
The PHP extension Imagick or Gmagick must be installed on the server to enjoy this feature.
```

File: docs/manual/article-management/content-elements/file-elements.en.md [hunk 2]
```diff
@@ -64,7 +134,48 @@ The special feature of Contao is that these download links also work with protec
...
+**Ignore files without meta data:** If no metadata for the appropriate page language has been entered for the files, 
+they are not displayed when activated.
+
+
+{{< version "4.13" >}}
+
+### Preview settings
+
+{{% notice info %}}
+The PHP extension Imagick must be installed on the server to enjoy this feature.
```
Comment (ausi) [inline docs/manual/article-management/content-elements/file-elements.en.md]:
```suggestion
The PHP extension Imagick or Gmagick must be installed on the server to enjoy this feature.
```

File: docs/manual/article-management/content-elements/file-elements.de.md [hunk 3]
```diff
@@ -29,6 +35,50 @@ erstellen. Weitere Informationen dazu erhältst im Abschnitt [Dateiverwaltung](.
 
 **Link-Titel:** Der Link-Titel wird als `title`-Attribut im HTML-Markup eingefügt.
 
+
+{{< version "4.13" >}}
+
+### Vorschau-Einstellungen
+
+{{% notice info %}}
+Die PHP-Erweiterung Imagick muss auf dem Server installiert sein, um in den Genuss dieser Funktion zu kommen.
```
Comment (ausi) [inline docs/manual/article-management/content-elements/file-elements.de.md]:
```suggestion
Die PHP-Erweiterung Imagick oder Gmagick muss auf dem Server installiert sein, um in den Genuss dieser Funktion zu kommen.
```

File: docs/manual/article-management/content-elements/file-elements.de.md [hunk 4]
```diff
@@ -77,6 +143,51 @@ Dateien.
 **Dateien ohne Metadaten ignorieren:** Wenn bei den Dateien keine Metadaten zur passenden Seitensprache eingepflegt 
 wurden, werden sie bei der Aktivierung nicht angezeigt.
 
+
+{{< version "4.13" >}}
+
+### Vorschau-Einstellungen
+
+{{% notice info %}}
+Die PHP-Erweiterung Imagick muss auf dem Server installiert sein, um in den Genuss dieser Funktion zu kommen.
```
Comment (ausi) [inline docs/manual/article-management/content-elements/file-elements.de.md]:
```suggestion
Die PHP-Erweiterung Imagick oder Gmagick muss auf dem Server installiert sein, um in den Genuss dieser Funktion zu kommen.
```

**Review by ausi — APPROVED** [review]
❤️

File: docs/manual/article-management/content-elements/file-elements.en.md (hunk 1 above)
Comment (fritzmg) [reply docs/manual/article-management/content-elements/file-elements.en.md]: Doesn't ImageMagick itself also have some minimum requirements?

Comment (ausi) [reply docs/manual/article-management/content-elements/file-elements.en.md]: Which formats are supported depends on the configuration of imagemagick, but it is probably best to solve this by showing the supported formats as suggested in https://github.com/contao/contao/issues/4122

**Pull request merged**